integration/scripts/integration.sh

```diff
@@ -82,7 +82,20 @@ run_integration_tests() {
     bash -c "${DC} exec -T xpu-spdk /usr/local/bin/identify    -r 'traddr:10.129.129.4 trtype:TCP adrfam:IPv4 trsvcid:4420'"
     bash -c "${DC} exec -T spdk-target /usr/local/bin/perf     -r 'traddr:10.129.129.4 trtype:TCP adrfam:IPv4 trsvcid:4420' -c 0x1 -q 1 -o 4096 -w randread -t 10"
     bash -c "${DC} exec -T xpu-spdk /usr/local/bin/perf         -r 'traddr:10.129.129.4 trtype:TCP adrfam:IPv4 trsvcid:4420' -c 0x1 -q 1 -o 4096 -w randread -t 10"
-    bash -c "${DC} up example-storage-client example-network-client"
+    NETWORK_CLIENT_NAME=$(${DC} ps | grep example-network-client | awk '{print $1}')
```

Comment:
you can use `${DC} ps example-network-client` instead of grep

Reply:
That doesn't work well because it prints header information as well.  Newer versions of docker compose have better support for filtering and formatting, but it doesn't work universally.

Reply:
ok